the-depot/Services/ReservationService.cs

```diff
@@ -106,11 +106,21 @@ public static void ScanCodeToAttend(Tour tour, string message)
 
         public static Reservation? AddReservation(int dayKey_Id, int tour_Id, out string addResponse)
         {
+            var tour = TourRepository.Get(tour_Id);
+            var attendeesCount = ReservationRepository.FindByTour(tour_Id).Count();
+            var reservations = ReservationRepository.All();
```

Comment:
reserveringen zijn al opgehaald voor de count, en dan hoef je ook niet voor die tour te checken in de if